src/main/java/melky/resourcepacks/SpriteOverride.java

```diff
@@ -160,6 +165,9 @@
 	SCROLLBAR_THUMB_BOTTOM(SpriteID.SCROLLBAR_THUMB_BOTTOM, Folder.SCROLLBAR),
 	SCROLLBAR_THUMB_MIDDLE_DARK(SpriteID.SCROLLBAR_THUMB_MIDDLE_DARK, Folder.SCROLLBAR),
 
+    WINDOW_CLOSE_BUTTON(SpriteID.WINDOW_CLOSE_BUTTON, Folder.OTHER),
+	WINDOW_CLOSE_BUTTON_HOVERED(SpriteID.WINDOW_CLOSE_BUTTON_HOVERED, Folder.OTHER),
```

Comment:
These both should be moved between `COMPASS` and `WINDOW_CLOSE_BUTTON_RED_X`. Also the `WINDOW_CLOSE_BUTTON`, needs to use a single tab for alignment rather than spaces